rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -84,13 +98,19 @@ install -m 644 launcher/sdw_notify/*.py %{buildroot}/opt/securedrop/launcher/sdw
 install -m 644 launcher/sdw_notify/*.py %{buildroot}/srv/salt/launcher/sdw_notify/
 install -m 644 launcher/sdw_util/*.py %{buildroot}/opt/securedrop/launcher/sdw_util/
 install -m 644 launcher/sdw_util/*.py %{buildroot}/srv/salt/launcher/sdw_util/
+install -m 644 config.json.example %{buildroot}/usr/share/%{srcname}/
+install -m 644 README.md LICENSE %{buildroot}/%{_custom_docdir}/
+find %{buildroot} -type d -iname '*.egg-info' -print0 | xargs -0 -r rm -rf
```

Comment:
isn't this why we set `--skip-build` in setup.py for? can we replace this command by a `setup.py` parameter that would make it more maintainable?

Reply:
I cannot find any python/setuptools invocation that will entirely skip creation of the "egg" directory (or zip archive). If you can, I would love to see it!

Reply:
No changes made here—I'm still unable to skip the egg creation dir entirely, but open to suggestions. I suspect if it's possible it'd require a newer version of setuptools than what Debian Stable provides. Possible, but would prefer to defer for now.